panel/command/serve.py

```diff
@@ -182,6 +182,12 @@ class Serve(_BkServe):
             action  = 'store_true',
             help    = "Whether to add an admin panel."
         )),
+        ('--admin-panel-name', dict(
+            metavar = "KEY=VALUE",
+            nargs   = '+',
+            help    = "Name to use for the admin panel.",
+            default = '/admin'
+        )),
```

Comment:
I'd call it `--admin-endpoint` for consistency and use `action = 'store', type=str`

Reply:
done